index.rst

```diff
@@ -6,65 +6,110 @@
 
+    .. figure:: /_static/mounted_exhaust.png
+       :name: mounted_exhaust
+       :target: ../_images/mounted_exhaust.png
+       :alt: mounted_exhaust
+       :scale: 10 %
+
+    .. figure:: /_static/exhaust_tube.png
+       :name: exhaust_tube
+       :target: ../_images/exhaust_tube.png
+       :alt: exhaust_tube
+       :scale: 10 %
+
 There are two cables that come from the WLS: one for commanding the power level and the other to receive status and error messages. 
 
-Documentation on the White Light Source can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5178>`__
+Further documentation on the White Light Source can be found on Docushare `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5178>`__
 
 Chiller
----------
-The chiller is used to cool the water cell in the white light source. This uses a mixture of ethanol and water (50/50)?? to carry heat from a shell surrounding the main water cell. The ethanol helps the water in these tubes from freezing. It does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. The control is on the water coming back to the chiller so there is likely to be some delay in large temperature swings. The temperature should be set to 25C (TBC).
+-------
+The chiller is used to cool the water cell in the white light source. This uses a mixture of methanol and water (10/90) to carry heat from a shell surrounding the main water cell. The ,ethanol helps the water in these tubes from freezing. The chiller does not actually pump the water in and out of the cell, but rather carries heat from tubes surrounding the water cell. The control is on the water returning to the chiller so there is likely to be some delay for large temperature swings. The temperature should be set to 25C.
 
-The chiller is controlled via connected to a serial device server (RS-232 to ethernet). The serial device server must run at 9600 baud. 
+The chiller is controlled via a serial device server (RS-232 to ethernet). The serial device server must run at 9600 baud. 
 
-The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. The white light source cannot be turned on until the chiller is running. 
+The white light source cannot be turned on until the chiller is running. The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. 
 
 The fluid level of the chiller should be checked occasionally. 
 
 The chiller documentation on Docushare can be found `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5177>`__
 
+.. note:: The flow of water from the chiller must be kept below 8 GPM. You can read this on the flow meter. To constrict the flow, use the nozzles on the tubes sitting beside the chiller
```

Comment:
Do you want to add anything about how the speed of the pump should always be at the minimum (or wherever you set it?) That was pretty critical to avoid leaks.

Reply:
yes, added. this.